mmv1/third_party/terraform/services/netapp/resource_netapp_backup_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package netapp_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccNetappbackup_netappBackupFull(t *testing.T) {
```

Comment:
Please extend this to make an [update test](https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test)

Reply:
Done